west.yml

```diff
@@ -100,7 +100,7 @@ manifest:
       revision: 41132e9220f8bc1223084975350c5e5f3b492afe
       path: tools/net-tools
     - name: hal_nxp
-      revision: b916bca1d5976b0157be80b326e3bb46ef605286
+      revision: pull/81/head
```

Comment:
I'll let someone else decide on policy, but it can't possibly be kosher to have a in-tree module pointed at a live pull request and not a committed tree.

Reply:
It's not. This is an intermediate step that allows us to verify both the HAL PR and the main zephyr repo PR in CI. Once the HAL PR gets merged, this will get updated to the new SHA before we merge this PR.